contracts/staking/IStakeManager.sol

```diff
@@ -5,15 +5,19 @@ contract IStakeManager {
   event Staked(address indexed user, uint256 indexed validatorId, uint256 indexed activatonEpoch, uint256 amount, uint256 total);
   event Unstaked(address indexed user, uint256 indexed validatorId, uint256 amount, uint256 total);
   event SignerChange(uint256 indexed validatorId, address indexed oldSigner, address indexed newSigner);
-
-  function stake(uint256 amount, address signer) external;
-  function stakeFor(address user, uint256 amount, address signer) external;
+  event ReStaked(uint256 indexed validatorId, uint256 amount, uint256 total);
+  event Jailed(uint256 indexed validatorId, uint256 indexed exitEpoch);
```

Comment:
Add following event:

```
event UnstakeInit(uint256 indexed validatorId, address indexed user, uint256 indexed amount, uint256 deactivationEpoch);
```

Also, please maintain parity between event arguments in `Staked` and `UnstakedInit`, like argument order: `user`, `validatorId`, `activationEpoch`/`deactivationEpoch`, `amount`.

Reply:
those are there in stakemanager will be moving into `IStakeManager.sol`